data/json/recipes/ammo/rifle.json

```diff
@@ -209,7 +209,10 @@
     "charges": 1,
     "using": [ [ "bullet_forming", 9 ], [ "ammo_bullet", 6 ] ],
     "components": [
-      [ [ "308_casing", 1 ] ],
+      [ 
+        [ "308_casing", 1 ],
+        [ "762_51_casing", 1 ]
+      ],
```

Comment:
Here I am assuming it is correct that one can create reloaded_308 ammo from 762_51_casing.

Reply:
Yeah, Id say thats fine since they are basically the same specifications irl.

Add them to the "casings" item group too.

Reply:
Oh, nevermind, looks like you did already.

Reply:
Thank you for the review. 

Are you saying to add them to here? (data/json/itemgroups/ammo.json) https://github.com/CleverRaven/Cataclysm-DDA/pull/24811/files#diff-ed6cfa7d7af668019c047bf58a69b2c8R407